tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+import testing.postgresql
+import console.settings
```

Comment:
imports should be pep8 compliant (three blocks, line seperating each block, first block stdlib imports, second installed modules, third local imports). In this case you want a space between l1 and l2. More info in the PEP here https://www.python.org/dev/peps/pep-0008/.